packages/roosterjs-editor-types/lib/enum/BulletListType.ts

```diff
@@ -32,4 +32,14 @@ export const enum BulletListType {
      * Bullet triggered by =>
      */
     UnfilledArrow,
```

Comment:
Add number value for each values, so that we clearly know how Min and Max are decided